### ISSUE TITLE
In Android 13 ImagePicker not working.

### ISSUE THREAD

**Issue by tayyabshafiq0602** [Dhaval2404/ImagePicker#320]
## Summary

In android 13 ImagePicker not working.

## Code to reproduce
In android 13 new permissions added please add this permission
<uses-permission android:name="android.permission.READ_MEDIA_IMAGES" />.

## Android version

13

## Installation method

implement through github 

## SDK version
33

**Comment by jstokke** [comment]
Yes, this is a real issue. You have to fall back to using PhotoPicker if the API version is 33 or higher. PhotoPicker can be used from API >=30

see https://developer.android.com/training/data-storage/shared/photopicker#kotlin for more info

Right now, using the library and targetting API 33 or above will give a permission error

**Comment by ArpitSharma32** [comment]
yes I am also facing the same in android 13, but the above solution is just for picking images. As we have integrated this lib choosing image from camera or gallery both and the output result can be cropped as per our requirement. so everything was managed by library itself.
If everything need to be done separately what will be the use of using it in the first place.

**Comment by jstokke** [comment]
The solution was the way the library would have to implement it, not as a workaround for the users of the library, but I have the feeling the author is no longer active.

**Comment by Faizanshah21** [comment]
Working fine on Android 13 for me. I was facing this issue myself. Upon some some debuggin it turns out it was a permission issue as I was using Dexter to handle permissions. Hope it helps anyone.

**Comment by wahyuafendi** [comment]
can u help me?

**Comment by hasanozn** [comment]
Android 13 does not work, although I give all permissions, there is no process when clicked. For example, the camera does not open and the gallery does not open.

**Comment by sharukhrs** [comment]
@tayyabshafiq0602, since the upgrade of API version 33, this library is not working anymore.
I was using it for more than a year now. It is very dificult to move to any other library since my app is heavily dependent on camera.
We appreaciate your revert on these queries.
